File: CodingBat/Python/Warmup_1/not_string.py
#Given a string, return a new string where not has been added to the front. However, if the string already begins with "not", return the string unchanged.
def not_string(str):
    if "not" in str:
        if str[0] == "n" and str[1] == "o" and str[2] == "t":
            return str
    return "not "+str

File: CodingBat/Java/Warmup_1/notString.java
public class notString{
    // Given a string, return a new string where "not" has been added to the front. However, if the string already begins with "not", return the string unchanged. Note: use .equals() to compare 2 strings.
    
    public static String notString(String str){
        if (str.length()>=3){
        if ((str.charAt(0)=='n')&&((str.charAt(1)=='o'))&&((str.charAt(2)=='t' ))){
            return str;
        }}
        return "not "+str;
    }
    public static void main(String[] args) {
        
    }
}